README.md

```diff
@@ -42,6 +42,18 @@ We really appreciate every possible kind of contributions. Pull-requests are gre
 
 Thanks to [Dakota Hawkins](https://github.com/dakotahawkins) for contributions.
 
+## Troubleshooting
+
+The first step to troubleshooting is to enable logging for the extension. This can be found in the settings for SourceTrail in `Tools` -> `Options` -> `Sourcetrail`
```

Comment:
Even though it may be tempting to write "SourceTrail" (many people actually do that), please change this to "Sourcetrail"